internal/namespaces/instance/v1/custom_server_create.go

```diff
@@ -231,22 +237,31 @@ func instanceServerCreateRun(ctx context.Context, argsI interface{}) (i interfac
 		serverReq.Image = args.Image
 	}
 
-	getImageResponse, err := apiInstance.GetImage(&instance.GetImageRequest{
-		Zone:    args.Zone,
-		ImageID: serverReq.Image,
-	})
-	if err != nil {
-		logger.Warningf("cannot get image %s: %s", serverReq.Image, err)
-	}
+	var (
+		getImageResponse *instance.GetImageResponse
+		serverType       *instance.ServerType
+	)
+	if args.Image != "none" {
+		getImageResponse, err := apiInstance.GetImage(&instance.GetImageRequest{
```

Comment:
`getImageResponse` is shadowing the one created in previous context.

```suggestion
		var err error
		getImageResponse, err = apiInstance.GetImage(&instance.GetImageRequest{
```